testkit-java/src/main/java/com/akkaserverless/javasdk/testkit/ActionResult.java

```diff
@@ -39,7 +39,9 @@
   /** @return true if the call was async, false if not */
   boolean isAsync();
 
-  CompletableFuture<Action.Effect<T>> getAsyncEffect();
+  CompletionStage<Action.Effect<T>> getAsyncEffect();
```

Comment:
Should be `CompletionStage<ActionResult<T>>`

Reply:
right. done, now